internal/worker/dbaccessor/doc.go

```diff
@@ -1,51 +1,76 @@
-// Copyright 2023 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
+// Copyright 2023 Canonical Ltd. Licensed under the AGPLv3, see LICENCE file for
```

Comment:
Pretty much everywhere else (though not in the `changestream` doc above), this is 2 lines - the copyright notice and the license.

Reply:
Fixed. (Feels like something that got lost when I started using automatic //-wrapping in my editor.)